pkg/apigw/filter/prefilter.go

```diff
@@ -89,17 +89,19 @@ func (h header) Handler() types.HandlerFunc {
 	return func(rw http.ResponseWriter, r *http.Request) error {
 		var (
 			ctx = r.Context()
+			vv  = map[string]interface{}{}
```

Comment:
Why are we solving this with `map[string]interface`? why not http.Header that can be converted into `expr.KVV`?